padertorch/contrib/examples/pit/__init__.py

```diff
@@ -0,0 +1,49 @@
+from .model import PermutationInvariantTrainingModel
+
+MAKEFILE_TEMPLATE_TRAIN = """
```

Comment:
Can you remove the code from the `__init__.py`?
It is always confusing, when you search the code.
Most of the time you only find the imports in the `__init__.py`

Reply:
I moved the templates to an additional file.